src/tests/components/main.test.js

```diff
@@ -0,0 +1,103 @@
+import {
+  fireEvent,
+  getByTestId,
+  render,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MainSection } from '../../components/MainSection';
+import testResponse from '../mocks/testResponse.json';
+
+describe('Test for Main section', () => {
+  test('should have a submit button', () => {
+    render(<MainSection />);
+
+    const submitButton = screen.getByRole('button', { name: /search/i });
+    expect(submitButton).toBeInTheDocument();
+  });
+
+  test('should have an image on the main section', async () => {
+    render(<MainSection />);
+
+    const imageEl = await screen.findByRole('img');
+    expect(imageEl).toBeInTheDocument();
```

Comment:
I think you can add an `expect` to test the `imageEl` element has `src` property equal to  the `testResponse`